packages/web3/src/contract/events.ts

```diff
@@ -40,11 +41,22 @@ export class EventSubscription extends Subscription<node.ContractEvent> {
     return this.fromCount
   }
 
+  private async getEvents(start: number): Promise<ContractEvents> {
+    try {
+      return await web3
+        .getCurrentNodeProvider()
+        .events.getEventsContractContractaddress(this.contractAddress, { start })
+    } catch (error) {
```

Comment:
If one subscribes to events starting from index 0 when the contract has no events, the latest full node will directly return `Contract events of contract-address not found`. Additionally, calling `getContractEventCount` will also return the error `Current events count for contract contract-address not found`. To avoid making an additional call to `getContractEventCount`, a catch was added here to handle this error.